src/utils/canvasSize.js

```diff
@@ -0,0 +1,162 @@
+/* eslint-disable no-console */
```

Comment:
Can we clean up this code to reduce usage of `eslint-disable`?

Reply:
Yes should clean up the code.  I was scare to break the code :D